tests/test_parser_error.py

```diff
@@ -15,15 +15,15 @@
 
 
 @irdl_op_definition
-class UnkownOp(IRDLOperation):
-    name = "unknown"
+class Unknown(IRDLOperation):
```

Comment:
```suggestion
class UnknownOp(IRDLOperation):
```
This change seems unnecessary